app/assets/stylesheets/views/_covid.scss

```diff
@@ -15,27 +15,40 @@ $c19-landing-page-header-background: govuk-colour("blue");
 
 .global-bar-present {
   @include govuk-media-query($from: desktop) {
-    .covid__page-header {
+    .covid__breadcrumb-wrapper {
       border-top: solid govuk-spacing(2) $c19-landing-page-header-background;
     }
   }
 
   @include govuk-media-query($until: desktop) {
-    .covid__page-header {
+    .covid__breadcrumb-wrapper {
       margin-top: govuk-spacing(-3);
     }
   }
 
-  .covid_page_header {
+  .covid__breadcrumb-wrapper {
     border-top: none;
   }
 }
 
+.covid__breadcrumb-wrapper {
+  background-color: $c19-landing-page-header-background;
+  border-top: solid govuk-spacing(2) $covid-yellow;
+  overflow: auto; // become a block formatting context to prevent bleeding the margins
+}
+
 .covid__page-header {
   margin-bottom: govuk-spacing(8);
   background-color: $c19-landing-page-header-background;
-  border-top: solid govuk-spacing(2) $covid-yellow;
   color: govuk-colour("white");
+
+  @include govuk-media-query($until: "tablet") {
```

Comment:
Just a nit-pick: we could combine this media query and the following one into:
```
padding-top: govuk-spacing(1);
  
@include govuk-media-query($from: "tablet") {
    padding-top: govuk-spacing(4);
  }
```

Reply:
I'll apply your suggestion shortly.

Reply:
Applied.